app/screens/post_options/post_options.js

```diff
@@ -205,56 +225,21 @@ export default class PostOptions extends PureComponent {
         );
     };
 
-    getMyPostOptions = () => {
-        const actions = [
-            this.getEditOption(),
-            this.getFlagOption(),
-            this.getAddReactionOption(),
-            this.getCopyPermalink(),
-            this.getCopyText(),
-        ];
-
-        const {canDelete, canPin} = this.props;
-        if (canPin) {
-            actions.splice(2, 0, this.getPinOption());
-        }
-        if (canDelete) {
-            actions.push(this.getDeleteOption());
-        }
-
-        return actions.filter((a) => a !== null);
-    };
-
-    getOthersPostOptions = () => {
+    getPostOptions = () => {
```

Comment:
there is a reason why we have `getMyPostOptions` and `getOthersPostOptions` the options display order is different for the two, please revert this change.

Reply:
Sorry! I was trying to refactor and messed up. What are the orders supposed to be? Does it make sense to get rid of the splice and push and just have the order in the actions array? Something like this but with the proper order:
```
getOthersPostOptions = () => {
        const actions = [
            this.getAddReactionOption(),
            this.getReplyOption(),
            this.getCopyText(),
            this.getEditOption(),
            this.getFlagOption(),
            this.getPinOption(),
            this.getCopyPermalink(),
            this.getDeleteOption(),
        ];

        return actions.filter((a) => a !== null);
    };

    getMyPostOptions = () => {
        const actions = [
            this.getAddReactionOption(),
            this.getReplyOption(),
            this.getCopyText(),
            this.getEditOption(),
            this.getFlagOption(),
            this.getPinOption(),
            this.getCopyPermalink(),
            this.getDeleteOption(),
        ];

        return actions.filter((a) => a !== null);
    };

    getPostOptions = () => {
        const {isMyPost} = this.props;

        return isMyPost ? this.getMyPostOptions() : this.getOthersPostOptions();
    };
```

Reply:
that would make sense yes.

The order is in the [spec](https://docs.google.com/document/d/1CZwdcPXTP9I3XqTbz67jomJ912TQrWF8BUr1BioEPuU/). If you don't have access to the doc feel free to request access to it.

Reply:
Thank you for the information! I've requested access and will update the PR once I get it.